src/inputType/PinchInput.ts

```diff
@@ -13,13 +12,20 @@ export interface PinchInputOption {
   scale?: number;
   threshold?: number;
   inputType?: string[];
+  touchAction?: string;
 }
 
 /**
  * @typedef {Object} PinchInputOption The option object of the eg.Axes.PinchInput module
  * @ko eg.Axes.PinchInput 모듈의 옵션 객체
  * @param {Number} [scale=1] Coordinate scale that a user can move<ko>사용자의 동작으로 이동하는 좌표의 배율</ko>
  * @param {Number} [threshold=0] Minimal scale before recognizing <ko>사용자의 Pinch 동작을 인식하기 위해산 최소한의 배율</ko>
+ * @param {String[]} [inputType=["touch", "pointer"]] Types of input devices
+ * - touch: Touch screen
+ * - pointer: Mouse and touch <ko>입력 장치 종류
+ * - touch: 터치 입력 장치
+ * - pointer: 마우스 및 터치</ko>
+ * @param {String} [touchAction="none"] Value that overrides the element's "touch-action" css property. It is set to "none" to prevent scrolling during touch. <ko>엘리먼트의 "touch-action" CSS 속성을 덮어쓰는 값. 터치 도중 스크롤을 방지하기 위해 "none" 으로 설정되어 있다.</ko>
```

Comment:
Automatically it is pan-y when horizontal, pan-x when vertical, and none when both are known as hammer methods. have you checked?

Reply:
Checked the older version of PanInput, and touch-action is set to one of "pan-x", "pan-y", or "none" depending on the connected axis.
Also in PinchInput, it is set to "none" regardless of the axis it is connected with. And "none" when using both input types.

Changes in the most recent commit follows this, and the default value of `touchAction` is null in PanInput.
When `touchAction` is null by default, "touch-action" is set by direction of connected axis.

This is something still thinking about, Is it okay to use the state where the touch-action is automatically determined when `touchAction` is null? Or is there a better way than this?